src/components/playback-controls/_playback-controls.scss

```diff
@@ -5,13 +5,13 @@
   top: 50%;
   left: 50%;
   margin-top: -32px;
-  margin-left: -48px;
+  margin-left: -51px;
 }
 
 .player.playlist {
   .player-gui > .playback-controls,
   .center-playback-controls {
-    margin-left: -144px;
+    margin-left: -153px;
```

Comment:
`transform: translate(-50%,-50%)` instead of the hard coded margin?

Reply:
it doesn't work well in playlist controls. see 
![image](https://user-images.githubusercontent.com/17685520/85408246-f0d7ee80-b56c-11ea-81e1-7b31087ce6e7.png)
Since the size calculation performed before the `transform` applied. 
with `margin` it's part of the size calculation

Reply:
actually works fine with `display:flex` so changed. @dan-ziv

Reply:
see support matrix: https://caniuse.com/#feat=flexbox
Need to check IE11 specifically.

Reply:
`flex` supported in ie but still looking bad in ie11 so reverted to hardcoded margin

Reply:
actually works fine with `white-space: nowrap` @dan-ziv @OrenMe